js/apps/system/_admin/aardvark/APP/frontend/js/views/userManagementView.js

```diff
@@ -255,6 +263,11 @@
       tableContent.push(
         window.modalView.createPasswordEntry('newPassword', 'Password', '', false, '', false)
       );
+      if (frontendConfig.isEnterprise) {
+        tableContent.push(
+          window.modalView.createCheckboxEntry('newRole', 'Role', false, false, false)
```

Comment:
The signature for `createCheckboxEntry` is: `createCheckboxEntry: function (id, label, value, info, checked)`
In this call, we are passing a boolean `false` as value, but a few lines below we are passing a string `'active'` as value.
Is that intentional?